README.md

```diff
@@ -1,6 +1,20 @@
-# analytics-platform
-Parent repository for the MOJ Analytics Platform. See [Wiki](https://github.com/ministryofjustice/analytics-platform/wiki) for more information.
+# Analytical Platform
 
-You can see a listing of all the repositories which are part of the platform [here](https://github.com/ministryofjustice?utf8=%E2%9C%93&q=analytics-platform&type=&language=).
+This is the root repository for the MOJ Analytical Platform.
 
-For an introduction to what the platform does, see the [User Guidance](https://user-guidance.services.alpha.mojanalytics.xyz/introduction.html#what-is-the-analytical-platform).
+Repositories: The platform is all open source - see all the repositories here: https://github.com/search?q=topic%3Aanalytics-platform+org%3Aministryofjustice&type=Repositories Within the repos there are some config files containing secrets, which are included in encrypted form.
+
+## Index of documentation
+
+* [User Guidance](https://user-guidance.services.alpha.mojanalytics.xyz/#content)
+* Technical Architecture
+  * [Overview](https://github.com/ministryofjustice/analytics-platform-ops/tree/main/docs/architecture)
+  * [Architecture diagrams](https://github.com/ministryofjustice/analytics-platform-ops/tree/main/docs/architecture#Diagrams)
```

Comment:
The architecture diagrams don't seem to render.

Reply:
I have another PR for that page, so I'll check this there: https://github.com/ministryofjustice/analytics-platform-ops/pull/346

Reply:
Yep images display on that PR